setup/storage/minio.md

```diff
@@ -39,3 +40,16 @@ echo $MINIO_SECRET_KEY | hal config storage s3 edit --endpoint $ENDPOINT \
 
 hal config storage edit --type s3 
 ```
+
+# Docker container instalation
+```bash
```

Comment:
Could you add a new line after these headers? 

Also, please give a callout to the fact that this should be run inside the docker container only.

Reply:
Thanks!